AmplifyPlugins/Predictions/AWSPredictionsPlugin/Service/Predictions/AWSPredictionsService+Transcribe.swift

```diff
@@ -0,0 +1,100 @@
+//
+// Copyright 2018-2019 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+import AWSTranscribeStreaming
+import AWSPluginsCore
+
+extension AWSPredictionsService: AWSTranscribeStreamingServiceBehavior {
+
+    func transcribe(speechToText: URL, language: LanguageType?, onEvent: @escaping TranscribeServiceEventHandler) {
+
+        guard let audioData = try? Data(contentsOf: speechToText) else {
+
+            onEvent(.failed(.network(AWSTranscribeStreamingErrorMessage.badRequest.errorDescription,
+                                     AWSTranscribeStreamingErrorMessage.badRequest.recoverySuggestion)))
+                   return
+        }
+
+        let request: AWSTranscribeStreamingStartStreamTranscriptionRequest =
+            AWSTranscribeStreamingStartStreamTranscriptionRequest()
+        let languageCode = language?.toTranscribeLanguage()
+        if let languageCode = languageCode {
+            request.languageCode = languageCode
+        } else {
+            request.languageCode = predictionsConfig.convert.transcription?.language.toTranscribeLanguage() ?? .enUS
+        }
+        request.mediaEncoding = .pcm
+        request.mediaSampleRateHertz = 8_000
+
+        transcribeDelegate.connectionStatusCallback = { status, error in
+            if status == .connected {
+                let headers = [
+                    ":content-type": "audio/wav",
+                    ":message-type": "event",
+                    ":event-type": "AudioEvent"
+                ]
+                let chunkSize = 4_096
+                let audioDataSize = audioData.count
+                var currentStart = 0
+                var currentEnd = min(chunkSize, audioDataSize - currentStart)
+
+                while currentStart < audioDataSize {
+                    let dataChunk = audioData[currentStart ..< currentEnd]
+                    self.awsTranscribeStreaming.send(data: dataChunk, headers: headers)
+
+                    currentStart = currentEnd
+                    currentEnd = min(currentStart + chunkSize, audioDataSize)
+                }
+                self.awsTranscribeStreaming.sendEndFrame()
```

Comment:
I am still concerned about concurrent requests. An endFrame could be send while another request is in flight.